packages/dashboard/src/customization/propertiesSections/textSettings/text.tsx

```diff
@@ -22,15 +22,15 @@ const ButtonWithState: FC<ButtonWithStateProps> = ({
   ...others // passing other attributes like data-test-id
 }) => {
   return (
-    <div
+    <button
```

Comment:
aria-label so a screen reader knows what the button is for?

Reply:
I think those are added when the component is used. ie line 94 in text.tsx